.travis/should_deploy_from_master.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+Should we run the deploy step on this Travis PR?
+
+We skip doing the publish/deploy step on jobs that don't have any
+relevant changes compared to master.
+
+Exits with code 1 if we should deploy, 0 otherwise.
+"""
+
+import os
+import subprocess
+import sys
+
+from tooling import are_there_job_relevant_changes, changed_files
+
+
+if __name__ == '__main__':
+    subprocess.check_call(['git', 'fetch', 'origin'])
+
+    changed_files = changed_files('HEAD', 'master')
+    task = os.environ['TASK']
+
+    if task in [
```

Comment:
Do you think it would be better to have some config in each "task" to signify which steps are available? I.e. agree a spec for some project config that gets consumed here?